File: PRAKTIKUM 2/BAHASA PY/PRAK205_2310817220028_SheilaSabina.py
import math

print("Input")
variabel = input().split()

tinggi, pythagoras = map(int, variabel)

alas = int(math.sqrt(pythagoras**2 - tinggi**2))
keliling = int(tinggi + pythagoras + alas)
luas = int(1 / 2 * alas * tinggi)

print("\nOutput")
print(f'Alas = {alas} cm')
print(f"Tinggi = {tinggi} cm")
print(f"Keliling = {keliling} cm")
print(f"Luas = {luas} cm^2")

File: PRAKTIKUM 2/BAHASA C/PRAK205_2310817220028_SheilaSabina.c
#include <stdio.h>
#include <math.h>

void main(){
    float a, b, c, keliling, luas;

    printf("Input\n");
    scanf("%f %f", &a, &b);

    c = sqrt(pow(b, 2) - pow(a, 2));
    keliling = a + b + c;
    luas = a * c / 2;

    printf("\nOutput\n");
    printf("Alas = %.0f cm\n", c);
    printf("Tinggi = %.0f cm\n", a);
    printf("Keliling = %.0f cm\n", keliling);
    printf("Luas = %.0f cm^2", luas);

}